docker/entrypoint.sh

```diff
@@ -9,4 +11,23 @@ if [ -n "$METADATA_AGENT_URL" ]; then
     /etc/google-fluentd/google-fluentd.conf
 fi
 
-/usr/sbin/google-fluentd "$@"
+trap 'kill $pid' TERM INT
+
+# first arg is a flag.
+if [ "${1:0:1}" = '-' ]; then
+  /usr/sbin/google-fluentd "$@" &
+else
+  exec "$@" &
+fi
+
+pid=$!
+wait $pid
+exit_code="$?"
+wait $pid
+
+# Ignore 143, as that is the code when SIGINT kills the process.
+if [[ exit_code -eq "143" ]]; then
+  exit 0
+fi
+
+exit $exit_code
```

Comment:
What are those two characters at the end of line? Just some weird display issue with Github?

Reply:
I think that's saying that there's no new line at the end of the file.

Reply:
Ah, yeah... Apparently. Just hovered on it and saw the explanation. LOL